src/models/Applicant/Section/Repository.ts

```diff
@@ -2,13 +2,35 @@ import { Applicant } from "../Model";
 import { Section } from ".";
 import { omit } from "lodash";
 import { TSection } from "../Interface";
+import { Op } from "sequelize";
+import isEmpty from "lodash/isEmpty";
+import { Transaction } from "sequelize";
 
 export const SectionRepository = {
-  updateOrCreate: async (applicant: Applicant, section: TSection) => {
+  update: async (applicant: Applicant, sections: TSection[], transaction?: Transaction) => {
+    const sectionsUuid: string[] = [];
```

Comment:
este for se podría reemplazar con un 

sectionUuids = sections.map(section => await updateorcreate(section).uuid)